config/locales/server.en.yml

```
@@ -2313,7 +2313,7 @@ en:
     allow_all_users_to_flag_illegal_content: "Anonymous users will see information that they have to e-mail administrators to report illegal content. This setting takes precedence over 'flag post allowed groups'."
     email_address_to_report_illegal_content: "If left blank the default site admin email will be used."
     cooldown_hours_until_reflag: "How much time users will have to wait until they are able to reflag a post"
-    slow_mode_prevents_editing: "Does 'Slow Mode' prevent editing, after editing_grace_period?"
+    slow_mode_prevents_editing: "Does 'Slow mode' prevent editing, after editing_grace_period?"
```

```suggestion
    slow_mode_prevents_editing: "Does 'Slow mode' prevent editing, after 'editing grace period'?"
```

Also may as well update this for consistency on how we refer to settings in translations

---

The `'site setting name'` method is currently "most consistent" :P But we do have internal discussions on how to deal with this more generally and make it clear to translators that this shouldn't be translated